### PR TITLE
Reorganise and cleanup the documentation

### PR DESCRIPTION
**Pull request by rafenden**
## Description of change

Move most of the `README.md` contents to separate files and update outdated documentation.

### CODE REVIEW

**Comment by paulgain** [comment]
https://github.com/uktrade/data-hub-frontend/tree/cleanup-documentation#table-of-contents

not-quite

File: docs/Environment variables.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+| HELP_CENTRE_URL                    | Help centre url                                                                                                                                                                                         |
+| HELP_CENTRE_ANNOUNCMENTS_URL       | Help centre announcements url                                                                                                                                                                           |
+| HELP_CENTRE_API_FEED               | Help centre feed API url                                                                                                                                                                                |
+| HELP_CENTRE_FEED_API_TOKEN         | Help centre API token                                                                                                                                                                                   |
```
Comment (paulgain) [inline docs/Environment variables.md]: I think we should mention the addition of `ZEN_TICKETS_URL` which refers to the Matching of Data Hub companies with D&B companies. I realise this is temporary so maybe it makes sense to add all the details to a new file, once we have no need for this file we can simply delete it and remove the link.

## Matching Interface on Staging
To POST an email into Zendesk (Staging) first take a look at how the code makes an API call.
https://github.com/uktrade/data-hub-frontend/blob/master/src/apps/support/services.js#L34-L45

There are 3 required environment variables that need to be defined within `.env` 
1. `ZEN_TICKETS_URL` (Zendesk Staging endpoint)
2. `ZEN_EMAIL`
3. `ZEN_TOKEN`

## Matching Interface on Sandbox
`ZEN_TICKETS_URL`=http://localhost:8001/zendesk/tickets

Comment (rafenden) [reply docs/Environment variables.md]: Good point, the file was updated.

File: docs/Sub-apps.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Sub-app structure
+# Sub-app
```
Comment (web-bert) [inline docs/Sub-apps.md]: Optional: should probably be plural

Comment (rafenden) [reply docs/Sub-apps.md]: Good point, code updated!

File: docs/Code review guidelines.md [hunk 3]
```diff
@@ -18,12 +19,14 @@
...
+
+- Follow branch naming conventions by prefixing your branch name with _feature/_, _bugfix/_, _removal/_, _test/_, _hotfix/_ or _release/_
 - Make sure commits are logical and atomic - each commit should include tests
 - Keep each commit small and deployable - each commit should ideally leave the develop branch in a releasable state
 - Use [imperative mood](https://git.kernel.org/pub/scm/git/git.git/tree/Documentation/SubmittingPatches?id=HEAD#n133) in commit message but it's okay to use past tense in the description
 
 #### PR Hygiene
+
 - Make sure your PR is atomic and doesn't solve multiple problems at the same time
 - Keep your PR small and deployable - each PR **must** leave the develop branch in a releasable state
```
[the remark this answers is not in the document]
Comment (rafenden) [reply docs/Code review guidelines.md]: Thanks, updated.

File: docs/Code review guidelines.md [hunk 4]
```diff
@@ -18,12 +19,14 @@
 - Make sure CI build passes consistently without any flaky tests
 
 #### Commit Hygiene
-- Follow branch naming conventions by prefixing your branch name with *feature/*, *bugfix/*, *removal/*, *test/*, *hotfix/* or *release/*
+
+- Follow branch naming conventions by prefixing your branch name with _feature/_, _bugfix/_, _removal/_, _test/_, _hotfix/_ or _release/_
 - Make sure commits are logical and atomic - each commit should include tests
 - Keep each commit small and deployable - each commit should ideally leave the develop branch in a releasable state
```
[the remark this answers is not in the document]
Comment (rafenden) [reply docs/Code review guidelines.md]: Thanks, updated.

**Comment by rafenden** [comment]
> Looks good, is it worth adding a section about the new logging env vars that I added to help debugging?
@web-bert the `Environment variables.md` file will be removed in https://github.com/uktrade/data-hub-frontend/pull/2395

**Pull request merged**